plugins/provisioners/ansible/provisioner/base.rb

```diff
@@ -112,6 +112,13 @@ def generate_inventory_groups
           group_vars = {}
           inventory_groups = ""
 
+          # Verify if host range patterns exist and warn
+          if config.groups.any? do |gm|
+             gm.to_s[/(?:\[[a-z]:[a-z]\]|\[[0-9]+?:[0-9]+?\])/]
+          end
```

Comment:
I'd prefer to use a single line `{|gm| ... }` notation for the `.any?` branch condition, to make the `if ... end` more readable